src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -80,28 +80,44 @@ You can also apply this configuration when unsetting a user at logout:
 
 <PlatformContent includePath="enriching-events/unset-user" />
 
+<PlatformSection supported={["android"]}>
+
+<PlatformContent includePath="enriching-events/scopes/scope-synchronization" />
+
+</PlatformSection>
+
 To learn what useful information can be associated with scopes see
 [the context documentation](../context/).
 
+<PlatformSection notSupported={["android"]}>
+
 ## Local Scopes
 
-We also have support for pushing and configuring a scope in one go. This is typically called `with-scope` or `push-scope` which is also very helpful if you only want to send data with one specific event. In the following example we are using that function to attach a `level` and a `tag` to only one specific error:
+We also support pushing and configuring a scope within a single call. This is typically
+called `with-scope` or `push-scope` depending on the SDK, and is very helpful if
+you only want to send data for one specific event. In the following example we use
+`with-scope` to attach a `level` and a `tag` to only one specific error:
 
 <PlatformContent includePath="enriching-events/scopes/with-scope" />
 
-While this example looks similar to `configure-scope`, it's very different, in the sense that `configure-scope` actually changes the current active scope, all successive calls to `configure-scope`
-will keep the changes.
+While this example looks similar to `configure-scope`, it is actually very different.
+Calls to `configure-scope` change the current active scope; all successive calls
+to `configure-scope` will maintain previously set changes unless they are explicitly
+unset.
 
-While on the other hand using `with-scope` creates a clone of the current scope
-and will stay isolated until the function call is completed. So you can either
-set context information in there that you don't want to be somewhere else or not
-attach any context information at all by calling `clear` on the scope, while the
-"global" scope remains unchanged.
+`with-scope` on the other hand creates a clone of the current scope and the changes
+made will stay isolated within the `with-scope` callback function. This allows you to
+easily isolate pieces of context information to specific locations in your code or
+even call `clear` to briefly remove all context information.
 
-Keep in mind that `with-scope` will not capture any exceptions that happen inside its callback function,
-and every error that happens there will be silently ignored and *not* reported.
+<Alert level="info" title="Important">
+Any exceptions that occur within the callback function for `with-scope` will not be
+caught and all errors that occur will be silently ignored and *not* repoted.
+</Alert>
+
+</PlatformSection>
 
-<PlatformSection supported={["java"]} notSupported={["android"]}>
+<PlatformSection supported={["java", "android"]}>
```

Comment:
Oh! with [this](https://github.com/getsentry/sentry-docs/pull/4747#discussion_r841714836) did you mean that you wanted to remove the mention of Kotlin as well as the local scopes stuff?

Reply:
Yes, for Android, since there's a global hub/scope, it does not make sense to use that integration, so we can remove from the android page.

Reply:
Got it, that makes sense. As a counter point though, I was confused that it was mentioned for Java and not Android, which is why I added it to begin with. What do you think about mentioning somewhere in the Android setup docs that the Kotlin extensions aren't needed for Android?

Reply:
Not sure, the content is not for Android at all, it even says:

> Sentry's SDK for Java stores the scope and the context in a thread-local variable

Instead of Sentry's SDK for Android, which is the case for all the other Android integrations, I guess explaining something from another platform would also make confusion.

Reply:
@designedbyz would you like to address this point so we can merge this? thanks.

Reply:
Sorry, too many irons in the fire. I should be able to address either later today or early tomorrow.

Reply:
@marandaneto done in 10b558ec8

Reply:
Agree, The goal is that `sentry-kotlin-extensions` get expanded and add extension functions to be "Kotlin friendly", but right now it only has the `SentryContext` and this only makes sense for non-Android.

E.g. https://github.com/getsentry/sentry-java/pull/1886/files
This would make sense for Android too.

Reply:
I'd leave this specific part for another PR, this will get addressed automatically when we land the 1st Kotlin function in the `sentry-kotlin-extensions` library for Android, which is gonna be very soon :)
Thanks for the feedback.

Reply:
> Agree, The goal is that sentry-kotlin-extensions get expanded and add extension functions to be "Kotlin friendly", but right now it only has the SentryContext and this only makes sense for non-Android.

Got it, that makes sense. 

> E.g. https://github.com/getsentry/sentry-java/pull/1886/files
This would make sense for Android too.

Oooh, that would be handy. 

> I'd leave this specific part for another PR, this will get addressed automatically when we land the 1st Kotlin function in the sentry-kotlin-extensions library for Android, which is gonna be very soon :)
Thanks for the feedback.

Awesome! That sounds good to me!